apps/quick-learn-frontend/src/app/(Dashboard)/dashboard/teams/[member]/teamMemberDetails.tsx

```diff
@@ -171,7 +171,7 @@ const TeamMemberDetails = () => {
             {member?.first_name} {member?.last_name}
           </h1>
           <p className="text-sm text-gray-500">
-            ({allRoadmapCategories.length} {en.common.roadmaps},{' '}
+            ({member?.assigned_roadmaps?.length} {en.common.roadmaps},{' '}
```

Comment:
While this will work correctly, if assigned_roadmaps is missing in the response, it will display undefined. Therefore, it’s recommended to set a default value of 0 to display in such cases.